source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/navigation.tsx

```diff
@@ -97,6 +97,14 @@ class Navigation extends React.Component<IProps & i18nProps> {
           </RequireRole>
         ) }
 
+        <RequireRole roleName={ROLE.SuperAdmin}>
+          <MenuItem
+            name={t('sidebar.adminSettings')}
+            to={`/admin/settings/organizations`}
```

Comment:
directly to settings/organization? not /admin? 

por que?

Reply:
Because right now there is no admin just organizations, perhaps wheb we had a basic config page we could change this

Reply:
all nested routes should have the following structure:
```
/{top-level-route}
  /{index}/ (exact match)
  /{nested-route-a}/
  /{nested-route-b}/
    /{index}/ (exact match)
```

Reply:
updated